app/src/main/java/org/simple/clinic/overdue/Appointment.kt

```diff
@@ -0,0 +1,82 @@
+package org.simple.clinic.overdue
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.PrimaryKey
+import android.arch.persistence.room.Query
+import com.squareup.moshi.Json
+import io.reactivex.Flowable
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.util.RoomEnumTypeConverter
+import org.threeten.bp.Instant
+import org.threeten.bp.LocalDate
+import java.util.UUID
+
+@Entity(tableName = "Appointment")
+data class Appointment(
+    @PrimaryKey val id: UUID,
+    val patientId: UUID,
+    val facilityId: UUID,
+    val date: LocalDate,
+    val status: Status,
+    val statusReason: StatusReason,
+    val syncStatus: SyncStatus,
+    val createdAt: Instant,
+    val updatedAt: Instant
+) {
+
+  enum class Status {
+
+    @Json(name = "scheduled")
+    SCHEDULED,
+
+    @Json(name = "cancelled")
+    CANCELLED,
+
+    @Json(name = "visited")
+    VISITED;
+
+    class RoomTypeConverter : RoomEnumTypeConverter<Status>(Status::class.java)
+  }
+
+  enum class StatusReason {
+
+    @Json(name = "not_called_yet")
+    NOT_CALLED_YET,
+
+    @Json(name = "not_responding")
+    PATIENT_NOT_RESPONDING,
+
+    @Json(name = "moved")
+    MOVED,
+
+    @Json(name = "dead")
+    DEAD;
+
+    class RoomTypeConverter : RoomEnumTypeConverter<StatusReason>(StatusReason::class.java)
+  }
+
+  @Dao
+  interface RoomDao {
```

Comment:
Should we start writing tests for the SQL queries that we run?

Reply:
Will they not get tested as part of other tests? Like `AppointmentSyncAndroidTest` in this case